consul/src/main/scala/io/buoyant/consul/v1.scala

```diff
@@ -44,7 +49,7 @@ package object v1 {
     Try(mapper.readValue[T](bytes, begin, end - begin))
   }
 
-  object Api {
+  object CatalogApi {
```

Comment:
Could we refactor this into an `Api` base trait and have `CatalogApi` and `KvApi` inherit from it?  Could also split these into 3 files (Api, CatalogApi, KvApi).